astropy/convolution/tests/test_kernel_class.py

```diff
@@ -32,7 +32,7 @@
                 MexicanHat1DKernel, Tophat2DKernel, AiryDisk2DKernel, Ring2DKernel]
 
 
-NUMS = [1, 1., np.float(1.), np.float32(1.), np.float64(1.)]
+NUMS = [1, 1., np.float32(1.), np.float64(1.)]
```

Comment:
This is one of the cases where I wasn't sure what was intended. Given that `np.float(1.)` is identical to `1.` I simply removed it.

Reply:
Agreed